### ISSUE TITLE
Ideas for improvement

### ISSUE THREAD

**Issue by humdrumoflife** [leafo/sightreading.training#25]
Wow, love this app. One thing to improve, is to not have previous notes cut off or disappear, as we are sight reading it is useful to see the note we previously put in so we cna look at intervals etc, but it disappears.

Thanks

**Comment by mredivo** [comment]
It would be important to make this feature optional, because looking ahead (instead of back at what you already played) is one of the things being learned.

**Comment by mredivo** [comment]
> well it is no different to looking at a sheet of music

Absolutely, and that's of course the "real life" situation. Having the notes disappear is a training situation, which some find helpful. That's the reason for my suggestion to have a switch to toggle the behaviour, instead of simply eliminating it.
